Resources/Prototypes/SS220/Entities/Mobs/NPCs/animals.yml

```
@@ -0,0 +1,107 @@
+- type: entity
+  name: Охранная собака
```

```suggestion
  name: охранная собака
```
С маленькой буквы, это не имя собственное